selfdrive/car/subaru/values.py

```diff
@@ -251,6 +251,7 @@ class SubaruCarInfo(CarInfo):
       b'\x00\x00eq\x1f@ "',
       b'\x00\x00eq\x00\x00\x00\x00',
       b'\x00\x00e\x8f\x00\x00\x00\x00',
+      b'\x00\x00e\x8f\x1f@ )',
```

Comment:
this one is a duplicate!
![Screenshot 2023-04-26 at 5 25 02 PM](https://user-images.githubusercontent.com/4038174/234729607-abdc47db-02cd-4b53-a0f8-e30ffd3674c5.png)